tanjun/commands.py

```diff
@@ -125,22 +129,31 @@ def bind_component(self, component: traits.Component, /) -> None:
     async def execute(
         self, ctx: traits.Context, /, *, hooks: typing.Optional[typing.MutableSet[traits.Hooks]] = None
     ) -> bool:
-        # TODO: parser
         try:
+            if self.parser is not None:
+                args, kwargs = await self.parser.parse(ctx)
+
+            else:
```

Comment:
Not sure I still support this use case I thought up of passing arg parser results to pre execution anymore, might remove